src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/ContentHelper.CoreClr.cs

```diff
@@ -14,13 +14,18 @@ internal static partial class ContentHelper
     {
         internal static Encoding GetEncoding(HttpResponseMessage response)
         {
-            string charSet = response.Content.Headers.ContentType.CharSet;
+                
+            //ContentType may not exist in response header.
```

Comment:
In the future it might not pass spelling verification. It would be nice to insert space after `//`
`// ContentType may not exist in response header.  Return null if not.`
And in below comments too.